module/framework.js

```diff
@@ -387,24 +422,12 @@ export async function getCategories({cache=true}={}) {
   // Get the category mapping
   const categories = await _getCategories({cache});
 
-  // Build the tree structure
```

Comment:
I think I need some reminder or explanation of why this code is safe to remove.

Reply:
Until now, we were recreating the whole tree ouselves, because we only had `parent_category` inside what we retrieved from WA.

But now, WA also gives `children.categories` inside its data. See `copyForSort`.
When we did this, we search for something like a `position` attribute inside what we retrieved. We now has instead a whole copy of its children.

Those copy will remain for what they are. We will only store one instance of each category in our tree.
But it will help us sort children for each category

> Short answer : Previous method lacked the ability to sort children. So it has been replaced by this one.

Reply:
This makes sense, it's good to know that the WA api is providing us more data and we have to make fewer decisions on our end.